applications/tasks.py

```diff
@@ -18,3 +28,36 @@ def populate_interviews_in_jobma(application_id):
     """Create an interview in Jobma and update our models with a link to the Jobma interview"""
     application = BootcampApplication.objects.get(id=application_id)
     api.populate_interviews_in_jobma(application)
+
+
+@app.task
+def refresh_pending_interview_links():
+    """ Recreate old pending interviews """
+    now = now_in_utc()
+    cutoff_date = now - timedelta(days=settings.JOBMA_LINK_EXPIRATION_DAYS)
+    for submission in (
+        ApplicationStepSubmission.objects.select_related("bootcamp_application")
+        .exclude(
+            Q(
+                bootcamp_application__state__in=(
```

Comment:
Can we move the list of app states into `constants.py` similar to `REVIEWABLE_APP_STATES`? We should also add a `.value` to the end of each enum